packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.js

```diff
@@ -131,40 +157,61 @@ StyledCloseButton.defaultProps = {
 
 const ErrorFormTooltip = ({
   onShown,
-  labelRef,
   dataTest,
   inputSize,
-  iconRef,
   children,
   shown,
-  isHelp = false,
+  referenceElement,
   inlineLabel,
+  isHelp = false,
   id,
 }: Props): React.Node => {
   const contentRef = React.useRef(null);
-  const tooltipRef = React.useRef<HTMLElement | null>(null);
-
-  const dimensions = useDimensions({
-    labelRef,
-    iconRef,
-    contentRef: tooltipRef,
-    children,
-    inlineLabel,
+  const { rtl } = useTheme();
+  const [tooltipRef, setTooltipRef] = React.useState(null);
```

Comment:
Why did you go for `useState` instead of `useRef` here? That's only needed for stuff like measuring.

Reply:
official docs are recommend to pass reference elements like that

Reply:
and it does not work properly with regular refs